tests/test_methods_and_attributes.cpp

```diff
@@ -335,6 +335,12 @@ TEST_SUBMODULE(methods_and_attributes, m) {
     py::class_<MetaclassOverride>(m, "MetaclassOverride", py::metaclass((PyObject *) &PyType_Type))
         .def_property_readonly_static("readonly", [](py::object) { return 1; });
 
+
```

Comment:
Add a comment here: `// test_overload_ordering` to reference the python test name.